### PR TITLE
Please add gfx803 to AMDGPU_TARGETS

### PR DESCRIPTION
**Pull request by xuhuisheng**
Please add gfx803 to AMDGPU_TARGETS, or it will break on gfx803. It wont cost too much time for compiling.
Thank you very much.

### CODE REVIEW

**Comment by xuhuisheng** [comment]
@doctorcolinsmith
Please have a look at this PR, thank you very much.

File: CMakeLists.txt [hunk 1]
```diff
@@ -64,9 +64,9 @@ endif()
 
 # Use target ID syntax if supported for AMDGPU_TARGETS
 if(TARGET_ID_SUPPORT)
-  set(AMDGPU_TARGETS gfx900:xnack-;gfx906:xnack-;gfx908:xnack- CACHE STRING "List of specific machine types for library to target")
+  set(AMDGPU_TARGETS gfx803;gfx900:xnack-;gfx906:xnack-;gfx908:xnack- CACHE STRING "List of specific machine types for library to target")
```
Comment (neon60) [inline CMakeLists.txt:67]: Please change this ```gfx803`` -> ```gfx803:xnack-```

**Comment by neon60** [comment]
The AMDGPU_TARGETS is not forced, which means you can change this during the cmake configuration. Is this solution not enough in our case?

```-DAMDGPU_TARGETS="gfx803;gfx900;gfx906;gfx908"```

File: CMakeLists.txt (hunk 1 above)
Comment (xuhuisheng) [reply CMakeLists.txt:67]: Thank you for responsing.
I followed other math projects like rocSPARSE, rccl. It left gfx803 without `xnack-`.

* https://github.com/ROCmSoftwarePlatform/rocSPARSE/commit/6cc282aaae2c28e880cf83e6934713282987d1a6
* https://github.com/ROCmSoftwarePlatform/rccl/commit/882d52ad7ec57e8055f71d978519c3352667fd5c

And <https://www.llvm.org/docs/AMDGPUUsage.html> said that gfx803 didnot support xnack, so it maybe neednot to config xnack for gfx803?

**Comment by xuhuisheng** [comment]
Hi @neon60 , We, who had gfx803 GPU, had just met this missing target issue on ROCm-3.9 and ROCm-3.10.
Please refer this issue : https://github.com/RadeonOpenCompute/ROCm/issues/1269
This issue caused tensorflow_rocm and pytorch crashed on ROCm-3.9 and ROCm-3.10 on very beginning of running.

Not everybody can compiling by himself. I am extremely graceful to you for help us to support gfx803 by offcial apt/rpm repository.

File: CMakeLists.txt (hunk 1 above)
Comment (xuhuisheng) [reply CMakeLists.txt]: @neon60 had changed `gfx803:xnack-`, please check.

**Comment by staticdev** [comment]
@stanleytsang-amd @doctorcolinsmith would you mind checking this one? It has the same idea of https://github.com/ROCmSoftwarePlatform/rocRAND/pull/167 and gfx803 customers would be very grateful you can also merge this PR. Same was already merged to [rocSPARSE](https://github.com/ROCmSoftwarePlatform/rocSPARSE/pull/213).

**Review by stanleytsang-amd — APPROVED** [review]

**Comment by neon60** [comment]
Conflict solution:
https://github.com/ROCmSoftwarePlatform/rocRAND/pull/170

**Comment by xuhuisheng** [comment]
> Conflict solution:
> #170

Thank you neon60, I will close this PR. Hope #170 could be available in the next version.

**Pull request closed without merging**